src/interface.ts

```diff
@@ -30,9 +33,9 @@ type RequireOnlyOne<T, Keys extends keyof T = keyof T> =
 export type PreLoadable = RequireOnlyOne<BasePreLoadable, 'require' | 'loader'>;
 
 export type EnhancedPreLoadable = {
-    cached: boolean;
```

Comment:
I understand that it's not used right now, but let's not remove it in this PR, please 🙏